R/fit_model.R

```diff
@@ -0,0 +1,42 @@
+#' Fit the RtGam model with {mgcv}
+#'
+#' Use the pre-prepared model dataset and formula. Supply warnings as needed
+#' @noRd
+fit_model <- function(data, formula, backend) {
```

Comment:
am I really not allowed to pass in any of my own args? What about supporting arg overwriting via .. and ModifyList? I realise it is slightly less opinionated but might open up a little more flexibility for very little code cost.

Reply:
This function isn't public?

Reply:
but you can pass ... all the way down so it doesn't need to be